src/mono/nuget/mono-packages.proj

```diff
@@ -9,7 +9,7 @@
     <ProjectReference Include="Microsoft.NET.Runtime.WebAssembly.Sdk\Microsoft.NET.Runtime.WebAssembly.Sdk.pkgproj" />
     <ProjectReference Include="..\wasm\templates\Microsoft.NET.Runtime.WebAssembly.Templates.csproj" />
     <ProjectReference Include="Microsoft.NET.Sdk.WebAssembly.Pack\Microsoft.NET.Sdk.WebAssembly.Pack.pkgproj" />
-
+    <ProjectReference Include="Microsoft.NET.WebAssembly.Webcil\Microsoft.NET.WebAssembly.Webcil.pkgproj" />
```

Comment:
This pkgproj doesn't exist, you'd need to create it

Reply:
Oh, I forgot to add the file!

Reply:
@thaystg if all you need is a nuget package then you don't need the .pkgproj (which is kinda deprecated), you can just set `<IsPackable>true</IsPackable>` in the .csproj.